ksql-workshop/ksql.commands

```diff
@@ -0,0 +1,17 @@
+--We need to identify fields in schema, even with Avro, because this is a script (https://github.com/confluentinc/ksql/issues/1031)
+CREATE STREAM ratings (RATING_ID BIGINT, USER_ID INTEGER, STARS INTEGER, ROUTE_ID INTEGER, RATING_TIME BIGINT, CHANNEL VARCHAR, MESSAGE VARCHAR) WITH (KAFKA_TOPIC='ratings', VALUE_FORMAT='AVRO');
+
+CREATE STREAM POOR_RATINGS AS SELECT * FROM ratings WHERE STARS <3 AND CHANNEL='iOS';
+
+CREATE STREAM CUSTOMERS_SRC (id BIGINT, first_name VARCHAR, last_name VARCHAR, email VARCHAR, gender VARCHAR, club_status VARCHAR, comments VARCHAR) WITH (KAFKA_TOPIC='customers', VALUE_FORMAT='JSON');
+
+CREATE STREAM CUSTOMERS_SRC_REKEY WITH (PARTITIONS=1, VALUE_FORMAT='AVRO') AS SELECT * FROM CUSTOMERS_SRC PARTITION BY ID;
+
+--We need to identify fields in schema, even with Avro, because this is a script (https://github.com/confluentinc/ksql/issues/1031)
+CREATE TABLE CUSTOMERS (id BIGINT, first_name VARCHAR, last_name VARCHAR, email VARCHAR, gender VARCHAR, club_status VARCHAR, comments VARCHAR) WITH (KAFKA_TOPIC='CUSTOMERS_SRC_REKEY', VALUE_FORMAT ='AVRO', KEY='ID');
```

Comment:
@ybyzek out of interest did you hit https://github.com/confluentinc/ksql/issues/1394 here (or in similar scripts)?

Reply:
@rmoff , no I did not.